source/codegen/metadata/visa/custom_header.mako

```diff
@@ -16,13 +16,15 @@ bytes_attrs = []
 for key, attr in attributes.items():
     attr_type = attr["type"]
     attr_name = f"VI_ATTR_{attr['name']}"
-    if ("MEM_BASE" in attr['name']
+    if ("_BASE" in attr['name']
+      or "_COUNT" in attr['name']
       or "MEM_SIZE" in attr['name']
-      or "WIN_BASE" in attr['name']
       or "WIN_SIZE" in attr['name']
     ):
         u32_attrs.append(f"{attr_name}_32")
         u64_attrs.append(f"{attr_name}_64")
+    elif "VI_ATTR_BUFFER" == attr['name']:
```

Comment:
This is necessary to prevent it from being put in with 'string' attributes which have a defined 256-character limit.